app/code/community/Meanbee/Royalmail/Model/Shipping/Carrier/Royalmail/Firstclasslettersignedfor.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to support@meanbee.com so we can send you a copy immediately.
+ *
+ * @category   Meanbee
+ * @package    Meanbee_Royalmail
+ * @copyright  Copyright (c) 2008 Meanbee Internet Solutions (http://www.meanbee.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Meanbee_Royalmail_Model_Shipping_Carrier_Royalmail_Firstclasslettersignedfor
+    extends Meanbee_Royalmail_Model_Shipping_Carrier_Royalmail_Abstract {
+
+    protected function getRates() {
+        $helper = Mage::helper('royalmail');
+        $rates = $this->_loadCsv('firstclass_letter_signedfor');
+
+        if ($helper->getWorldZone($this->_getCountry()) == 'gb') {
```

Comment:
Would be good to use constants for GB, e.g. `Meanbee_Royalmail_Helper_Data::WORLD_ZONE_GB`.